src/main/java/com/novi/backend_spring_boot_tech_it_easy_controller/BackendSpringBootTechItEasyControllerApplication.java

```diff
@@ -0,0 +1,15 @@
+package com.novi.backend_spring_boot_tech_it_easy_controller;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@SpringBootApplication
+@RequestMapping("/api")
```

Comment:
Deze annotatie mag boven de controller klasse. Deze annotatie geeft een  base pad aan al je mappings/endpoints. In dit geval is je base pad dus "/api" en zou je bijvoorbeeld voor je getAllTelevisions mapping het pad krijgen "localhost:8080/api/televisions"